### PR TITLE
DOC: Resampling code example

### PR DESCRIPTION
**Pull request by remi-braun**
- Code example of the equivalent of [rasterio's resampling](https://rasterio.readthedocs.io/en/latest/topics/resampling.html)

### CODE REVIEW

File: docs/examples/resampling.ipynb [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+   "source": [
+    "from rasterio.enums import Resampling\n",
+    "\n",
+    "# Reproject   \n",
+    "upscale_factor = 2\n",
+    "new_width = xds.rio.width * upscale_factor\n",
+    "new_height = xds.rio.height * upscale_factor\n",
+    "resampling=Resampling.bilinear\n",
+    "\n",
+    "xds_upsampled = xds.rio.reproject(xds.rio.crs, shape=(new_height, new_width))\n"
```
Comment (snowman2) [inline docs/examples/resampling.ipynb:106]: Don't forget to pass in `resampling` kwarg.

Comment (remi-braun) [reply docs/examples/resampling.ipynb:106]: Yes my bad !

File: docs/examples/resampling.ipynb [hunk 2]
```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example - Resampling\n",
+    "\n",
+    "API Reference for `rio.clip`:\n",
```
Comment (snowman2) [inline docs/examples/resampling.ipynb:9]: I see `rio.clip`, but it is for `rio.reproject`. If you copied from another example. I wonder if that is an issue there too?

File: docs/examples/resampling.ipynb (hunk 1 above)
Comment (snowman2) [reply docs/examples/resampling.ipynb]: Thanks for addressing that :+1:

File: docs/examples/resampling.ipynb (hunk 2 above)
Comment (remi-braun) [reply docs/examples/resampling.ipynb:9]: Rah sorry, this is me doing to many things in the same time... 😓 
I copied from clip, don't worry

Comment (snowman2) [reply docs/examples/resampling.ipynb]: Phew, thanks :sweat_smile:

**Comment by snowman2** [comment]
Thanks @remi-braun :+1:

**Pull request merged**